src/components/icons/svg/IconDoubleStarsFilled.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import { Path, Svg } from "react-native-svg";
+import { SVGIconProps } from "../Icon";
+const IconDoubleStarsFilled = ({ size, style, ...props }: SVGIconProps) => (
+  <Svg width={size} height={size} viewBox="0 0 16 17" style={style} {...props}>
+    <Path
+      fill="#5517E3"
```

Comment:
```suggestion
      fill="currentColor"
```
Better to leave the icon color-agnostic and set the color using the general color palette of the app (`IOColors`)

Reply:
addressed in [86ab979](https://github.com/pagopa/io-app-design-system/pull/139/commits/86ab9790177fe4ed7ef2c9560635c01f03991d49)